docs/APDB.md

```diff
@@ -0,0 +1,47 @@
+Alert Production Database (APDB) Schema
+=======================================
+
+The [apdb](./python/lsst/sdm_schemas/schemas/apdb.yaml) schema describes the Alert Production Database (APDB).
+This database is updated by the [Alert Production Pipeline](https://github.com/lsst/ap_pipe) and used for nightly [ap_verify](https://github.com/lsst/ap_verify) runs within continuous integration.
+Previous processing runs may differ from the current schema.
+The user-queryable Prompt Products Database (PPDB) is expected to have a very similar schema to the APDB.
+
+Schema Versioning
+-----------------
+
+The versioning system of the APDB is outlined in [DMTN-269](https://dmtn-269.lsst.io/).
+The deployed database schema is a product of the [schema in sdm_schemas](yml/apdb.yaml) and processing by tools in the [dax_apdb](https://github.com/lsst/dax_apdb) repository.
+Compatibility of the client executable with the actual database schema is determined by the version number in `apdb.yaml` and that which is stored in the database metadata.
+
+Backward compatibility in the context of the APDB implies that a client using an interface derived from a schema with a higher `MINOR` version number will be able to read and write into a database with a lower `MINOR` version.
```

Comment:
Hmm, I don't think I put it like that on the Confluence page. In APDB there is no "interface derived from a schema". There is just schema used by the client to run the queries and an actual schema in the database. It may be better to copy that page more or less verbatim, I think it provides clearer explanation of what compatibility means for APDB?

Reply:
I'm guessing that was a reference to `Apdb`, since I mentioned earlier that we never do direct queries.

Reply:
I attempted to reword this section so it was clearer and matched your text better.